CMakeLists.txt

```diff
@@ -168,6 +173,11 @@ else()
     set(RUNTIME_DIR_DEFAULT "${CMAKE_INSTALL_FULL_LOCALSTATEDIR}/run/sddm")
 endif()
 
+# x11
+if(BUILD_WITH_X11)
+    add_definitions(-DBUILD_WITH_X11)
```

Comment:
IMO definitions should always be 0 or 1, to avoid that a typo breaks stuff silently.

Reply:
Fixed.